src/223/test/kotlin/org/rust/ide/intentions/RsIntentionTestPlatformBase.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.intentions
+
+import com.intellij.codeInsight.intention.IntentionAction
+import org.rust.RsTestBase
+
+// BACKCOMPAT: 2022.2. Merge into `RsIntentionTestBase`
+abstract class RsIntentionTestPlatformBase : RsTestBase() {
+    protected fun checkPreviewAndLaunchAction(intention: IntentionAction, preview: String?) {
+        val text = myFixture.getIntentionPreviewText(intention)
+        check(text != null) { "No preview for intention ${intention.text}" }
+        myFixture.launchAction(intention)
+        assertEquals(intention.text, preview ?: myFixture.file.text, text)
```

Comment:
Let's add some message to `assertEquals`  to explain a developer what happened. Intention preview is quite a new feature, so a developer can just not know about it